### DISCUSSION TITLE
Shortcut to hide/show scroll bars

### DISCUSSION THREAD

**Discussion by 13R4F431** [sumatrapdfreader/sumatrapdf#2526] [Ideas]
Hi, I wonder if could be useful to include a shortcut for instance F7 to hide/show scroll bars, working like F8, F9, F11 control the visualization of the reader.

**Comment by kjk** [comment]
In [pre-release builds](https://www.sumatrapdfreader.org/prerelease) you can bind the command `CmdToggleScrollbars` to any shortcut you want. See https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

**Reply by 13R4F431** [reply]
Ok, I saw, I did. Thank you.
